swagger.json

```diff
@@ -5796,6 +5861,55 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "Contact": {
+            "type": "object",
+            "description": "Information about a contact.",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the contact",
+                    "format": "int64",
+                    "example": 123
+                },
+                "userId": {
+                    "type": "integer",
+                    "description": "User ID of the contact",
+                    "format": "int64",
+                    "example": 123
+                },
+                "orgId": {
```

Comment:
Do we need to return this? The caller should already know the organization.

Reply:
Removed the field.